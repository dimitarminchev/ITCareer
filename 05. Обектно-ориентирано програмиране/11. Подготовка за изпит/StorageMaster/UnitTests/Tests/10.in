RegisterStorage Warehouse DaWarehouse
RegisterStorage AutomatedWarehouse TheAutomatedWarehouse
RegisterStorage DistributionCenter TheDistributionCenter
RegisterStorage Invalid InvalidStorage
AddProduct Gpu 100
AddProduct HardDrive 100
AddProduct SolidStateDrive 100
AddProduct Ram 100
AddProduct Product 100
AddProduct Gpu 100
AddProduct HardDrive 100
AddProduct SolidStateDrive 100
AddProduct Ram 100
AddProduct Product 100
SelectVehicle DaWarehouse 0
LoadVehicle Gpu HardDrive Gpu HardDrive
SelectVehicle TheAutomatedWarehouse 0
LoadVehicle SolidStateDrive SolidStateDrive Ram Ram
SelectVehicle TheDistributionCenter 0
LoadVehicle Ram
SendVehicleTo DaWarehouse 0 TheDistributionCenter
SendVehicleTo DaWarehouse 0 TheDistributionCenter
SendVehicleTo DaWarehouse 1 TheDistributionCenter
GetStorageStatus DaWarehouse
GetStorageStatus TheDistributionCenter
GetStorageStatus TheAutomatedWarehouse
UnloadVehicle TheDistributionCenter 3
UnloadVehicle TheDistributionCenter 4
END
