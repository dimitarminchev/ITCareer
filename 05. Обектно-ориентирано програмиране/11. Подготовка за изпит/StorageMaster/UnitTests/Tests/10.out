Registered DaWarehouse
Registered TheAutomatedWarehouse
Registered TheDistributionCenter
Error: Invalid storage type!
Added Gpu to pool
Added HardDrive to pool
Added SolidStateDrive to pool
Added Ram to pool
Error: Invalid product type!
Added Gpu to pool
Added HardDrive to pool
Added SolidStateDrive to pool
Added Ram to pool
Error: Invalid product type!
Selected Semi
Loaded 4/4 products into Semi
Selected Truck
Loaded 4/4 products into Truck
Selected Van
Error: Ram is out of stock!
Sent Semi to TheDistributionCenter (slot 3)
Error: No vehicle in this garage slot!
Sent Semi to TheDistributionCenter (slot 4)
Stock (0/10): []
Garage: [empty|empty|Semi|empty|empty|empty|empty|empty|empty|empty]
Stock (0/2): []
Garage: [Van|Van|Van|Semi|Semi]
Stock (0/1): []
Garage: [Truck|empty]
Unloaded 3/4 products at TheDistributionCenter
Error: Storage is full!
TheDistributionCenter:
Storage worth: $300.00
DaWarehouse:
Storage worth: $0.00
TheAutomatedWarehouse:
Storage worth: $0.00
